Userland/Libraries/LibJS/Runtime/Uint8Array.cpp

```diff
@@ -729,7 +729,7 @@ DecodeResult from_base64(VM& vm, StringView string, Alphabet alphabet, LastChunk
                 ch = '+';
             }
             // iii. Else if char is "_", then
-            else if (ch == '-') {
+            else if (ch == '_') {
```

Comment:
Fix looks good, but please also add a test to Userland/Libraries/LibJS/Tests/builtins/TypedArray/Uint8Array.fromBase64.js that covers this case, since we've missed it.

Reply:
It took me ages, but I have finally managed to reproduce the bug from JS. Original tests not just didn't test `_` handling with base64url decoding, but also didn't test `lastChunkHandling: "strict"`, that is crucial as default Loose mode just triggers `AK::decode_base64url_into`, that doesn't fail